### ISSUE TITLE
Inventory: App Module Components

### ISSUE THREAD

**Issue by laurelthrash** [NCIOCPL/ncids#646]
## In order to build app modules consistent with NCI DS, designers need to identify the components needed to create app modules.
> **Estimate** *5* 

### Description:
Determined on ~Sept. 15 2022 that we will need to take an inventory of all components our app modules currently use. Once an inventory is complete, we'll then need to determine what of these components already exist in the USWDS or need to be created as custom NCIDS components (this will be done in a separate ticket).

### Expected Deliverables:
* An inventory of components needs for app modules (Mural Board), be sure to record the following
** Component name/type
** Component example screenshot
** Component use case/function
* Meet with Sarina before discussing with the larger team to ensure everything within the app modules is covered
* Discuss with broader team

### Additional Notes:

App modules for which a component review is necessary are below: 
| App Module | Prod Link |
-- | --
Clinical Trials Search | https://www.cancer.gov/about-cancer/treatment/clinical-trials/search
Listing Pages (Dynamic and Manual) | https://www.cancer.gov/about-cancer/treatment/clinical-trials/disease/breast-cancer/ & https://www.cancer.gov/about-cancer/treatment/clinical-trials/covid-19
*Dictionary of Cancer Terms | https://www.cancer.gov/publications/dictionaries/cancer-terms
*Genetics Dictionary | https://www.cancer.gov/publications/dictionaries/genetics-dictionary
Drug Dictionary | https://www.cancer.gov/publications/dictionaries/cancer-drug
Search Results | https://www.cancer.gov/search/results?swKeyword=cancer
R4R (Resources for Researchers) | https://www.cancer.gov/research/resources

</body>
</html>

### Results of Spike
*Enter the information, or links to any (non-sensitive) files with the findings here*

**Comment by blilianyu** [comment]
Here is the mural board with the inventory of components. https://app.mural.co/t/publicissapient7269/m/publicissapient7269/1667482049826/47ec82a82b7cabe54cd6b8d164aedcd1a909fc4e?sender=uf7d7cce75e812c2b5a084619
